postgres/changelog.d/16268.changed

```diff
@@ -0,0 +1 @@
+PostgreSQL: Add granted tag to `postgresql.locks` metric
```

Comment:
can we update the changelog to `.added`?

Reply:
Done.